src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Cohort/FileUploadValidateDataRequestToApiRequest.cs

```diff
@@ -3,23 +3,36 @@
 using SFA.DAS.ProviderCommitments.Infrastructure.OuterApi.Requests;
 using SFA.DAS.ProviderCommitments.Interfaces;
 using SFA.DAS.ProviderCommitments.Queries.BulkUploadValidate;
-using SFA.DAS.ProviderCommitments.Web.Extensions;
 using System.Threading.Tasks;
 
 namespace SFA.DAS.ProviderCommitments.Web.Mappers.Cohort
 {
     public class FileUploadValidateDataRequestToApiRequest : FileUploadMapperBase, IMapper<FileUploadValidateDataRequest, BulkUploadValidateApimRequest>
     {
-        public FileUploadValidateDataRequestToApiRequest(IEncodingService encodingService, IOuterApiService outerApiService) 
+        private readonly Authentication.IAuthenticationService _authenticationService;
+
+        public FileUploadValidateDataRequestToApiRequest(IEncodingService encodingService, IOuterApiService outerApiService, Authentication.IAuthenticationService authenticationService) 
             :base(encodingService, outerApiService)
-        { }
+        {
+            _authenticationService = authenticationService;
+        }
 
         public Task<BulkUploadValidateApimRequest> Map(FileUploadValidateDataRequest source)
         {
             var apiRequest = new BulkUploadValidateApimRequest();
             apiRequest.ProviderId = source.ProviderId;
             apiRequest.RplDataExtended = source.RplDataExtended;
             apiRequest.CsvRecords = ConvertToBulkUploadApiRequest(source.CsvRecords, source.ProviderId);
+
+            var userinfo = new ApimUserInfo()
+            {
+                UserDisplayName = _authenticationService.UserName,
+                UserEmail = _authenticationService.UserEmail,
+                UserId = _authenticationService.UserId,
+            };
+
+            apiRequest.UserInfo = userinfo;
+            
```

Comment:
I checked this code and because it's going via a mapper and inherits from ApimSaveDataRequest it will already populate the userinfo. So this code isn't required